src/bindings/html/view.js

```diff
@@ -41,13 +43,21 @@ export class View {
   formatEntity(id, args) {
     return this.ctx.formatEntity(this.service.languages, id, args);
   }
+
+  translateFragment(frag) {
+    return translateFragment(this, frag);
+  }
 }
 
 View.prototype.setAttributes = setL10nAttributes;
 View.prototype.getAttributes = getL10nAttributes;
-View.prototype.translateFragment = translateFragment;
 
-function onMutations(mutations) {
+export function init(langs) {
+  dispatchEvent(this.doc, 'supportedlanguageschange', langs);
+  return translateDocument(this, this.doc, langs);
```

Comment:
I don't like this. View.init should not send supportedlanguageschange, it probably shouldn't even translateDocument, unless you have a flag to not do that. It feels like it doesn't belong here.